README.md

```diff
@@ -18,27 +18,27 @@ git clone git@github.com:ONSdigital/dp-cli.git
 
 **Required:**
 
-The DP CLI uses Go Modules so requires a go version of **1.11** or later.
+The DP CLI uses Go Modules so requires a go version of **1.17** or later.
```

Comment:
You state 1.17 in this line, which may be fine, but you have updated all the following references of 1.17 to 1.18
Should this line also be 1.18 for consistency ?